spec/filters/dissect_spec.rb

```diff
@@ -3,6 +3,31 @@
 require "logstash/filters/dissect"
 
 describe LogStash::Filters::Dissect do
+  class LoggerMock
```

Comment:
@jordansissel @ph @andrewvc @talevy 
As a side note, in rspec we need a better abstraction for expectations for cross-cutting aspects like logging and metrics.
If tests like these expect exact logging and/or metrics - they are brittle.
I'm thinking we need a better way to set a less exact expectation.
If I get some +1s on this I will create a discuss issue.